### DISCUSSION TITLE
No user interaction?

### DISCUSSION THREAD

**Discussion by g003441** [grahampugh/erase-install#459] [General]
So whats the rational for not using the --very-insecure-mode flag?

Also how do we do it correctly? "/Library/Management/Erase-install/erase-install.sh --reinstall --credentials --very-insecure-mode" is still asking for the users password.

**Comment by grahampugh** [comment]
See https://github.com/grahampugh/erase-install/wiki/4.-Upgrading-or-reinstalling-macOS-without-wiping-the-system#--very-insecure-mode----credentials
